src/components/timeline/TimelineLockControl.svelte

```diff
@@ -49,7 +55,11 @@
 <button
```

Comment:
If I unlock activity dragging as a user with permission, then switch to a user without permission, then drag on an activity, I see this error and the UI locks up
![Uploading image.png…]()